tests/test_authentication_dummy.py

```diff
@@ -1,20 +1,38 @@
 import unittest
 
+from trytond.pool import Pool
+import trytond.tests.test_tryton
 from trytond.tests.test_tryton import ModuleTestCase
-from trytond.tests.test_tryton import suite as test_suite
+from trytond.tests.test_tryton import with_transaction
 
-from .test_login import LoginTestCase
+from . import factories
 
 
-class AuthenticationDummyTestCase(
-        ModuleTestCase,
-        LoginTestCase,
-):
+class AuthenticationDummyTestCase(ModuleTestCase):
     'Test Authentication Dummy module'
     module = 'authentication_dummy'
 
+    @with_transaction()
+    def test_login_dummy(self):
+
+        # GIVEN
+        User = Pool().get('res.user')
+        user = factories.User.create(
+            login='test',
+            password='good_pass'
+        )
+
+        # WHEN
+        login = User.get_login(
+            'test', 'bad_pass',
```

Comment:
Shouldn't the second parameter be an empty dict? The point of this module is that it will authenticate any user with 0 authentication proofs. That is without even asking for a password!

But if it was wanted to accept bad passwords, then I think the get_login call expects the second parameter to be a dict with magic words on the keys and authentication proofs on the values, be it passwords or OTP tokens and whatnot.

Obviously the test works because the module is correctly ignoring the second parameter altogether, but for completeness I'd rather write it like
```
login = User.get_login('test', {})
```
If a bad password was needed, such a call would be expected to raise an specific exception wrapping the magic key where the client should store the next required authentication proof, which is not :tada: